src/03-setup/linux.md

```diff
@@ -28,22 +26,25 @@ $ sudo dnf install \
 
 ``` console
 $ sudo pacman -S \
-  arm-none-eabi-gcc \
   arm-none-eabi-gdb \
   minicom \
   openocd
 ```
 
-- Other distros
+- `arm-none-eabi-gdb` for other distros
```

Comment:
We should probably also mention the possibility of using a multi-architecture gdb